internal/queue/score/score.go

```diff
@@ -0,0 +1,98 @@
+package score
+
+import (
+	"time"
+
+	"github.com/takenet/deckard/internal/dtime"
+)
+
+// TODO: keeping these as variables to get their reference and use less memory. Is this really necessary?
+// TODO: benchmark this
+var Max float64 = 9007199254740992
```

Comment:
Is there a need to use float instead of integers which could yield bigger numbers to be used as score ?

Reply:
We use this since our implementation was drived by redis' internal definitions. Scores for sorted sets are stored as float64:
```
The score values should be the string representation of a double precision floating point number. +inf and -inf values are valid values as well.
```

We can have bigger numbers than `9007199254740992` in a float representation, but it will be stored as a cientific notation without precision.